### PR TITLE
Governance integration

### LINKED ISSUE #109: Governance integration

### PR DESCRIPTION
**Pull request by ayushmishra2005**
resolves #109

### CODE REVIEW

File: runtime/parallel/Cargo.toml [hunk 1]
```diff
@@ -60,13 +67,13 @@ sp-block-builder                           = { git = 'https://github.com/parityt
...
 sp-offchain                                = { git = 'https://github.com/paritytech/substrate.git', version = '3.0.0', branch = 'polkadot-v0.9.1', default-features = false }
 sp-runtime                                 = { git = 'https://github.com/paritytech/substrate.git', version = '3.0.0', branch = 'polkadot-v0.9.1', default-features = false }
 sp-session                                 = { git = 'https://github.com/paritytech/substrate.git', version = '3.0.0', branch = 'polkadot-v0.9.1', default-features = false }
 sp-std                                     = { git = 'https://github.com/paritytech/substrate.git', version = '3.0.0', branch = 'polkadot-v0.9.1', default-features = false }
 sp-transaction-pool                        = { git = 'https://github.com/paritytech/substrate.git', version = '3.0.0', branch = 'polkadot-v0.9.1', default-features = false }
 sp-version                                 = { git = 'https://github.com/paritytech/substrate.git', version = '3.0.0', branch = 'polkadot-v0.9.1', default-features = false }
+static_assertions                          = "1.1.0"
```
Comment (GopherJ) [inline runtime/parallel/Cargo.toml:76]: let's stick to single quote

File: runtime/vanilla/Cargo.toml [hunk 2]
```diff
@@ -18,9 +18,10 @@ git     = 'https://github.com/paritytech/substrate.git'
 version = '4.0.0'
 
 [dependencies]
-codec       = { package = 'parity-scale-codec', version = '2.0.0', default-features = false, features = ['derive'] }
-hex-literal = { version = '0.3.1', optional = true }
-serde       = { version = '1', features = ['derive'], optional = true }
+codec                     = { package = 'parity-scale-codec', version = '2.0.0', default-features = false, features = ['derive'] }
+hex-literal               = { version = '0.3.1', optional = true }
+serde                     = { version = '1', features = ['derive'], optional = true }
+static_assertions         = "1.1.0"
```
Comment (GopherJ) [inline runtime/vanilla/Cargo.toml:24]: let's stick to single quote

**Comment by GopherJ** [comment]
@ayushmishra2005 I think it's better to do

git checkout origin/master Cargo.lock

then add your dependencies which may fix the MaxLocks issue

File: runtime/parallel/src/lib.rs [hunk 3]
```diff
@@ -630,6 +635,233 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = ();
 }
 
+parameter_types! {
+    pub const LaunchPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
```
Comment (kaichaosun) [inline runtime/parallel/src/lib.rs:639]: The period needs to reconsidered.

File: runtime/parallel/src/lib.rs [hunk 4]
```diff
@@ -630,6 +635,233 @@ impl pallet_multisig::Config for Runtime {
...
+    type ProposalBondMinimum = ProposalBondMinimum;
+    type SpendPeriod = SpendPeriod;
+    type Burn = Burn;
+    type BurnDestination = ();
+    type SpendFunds = Bounties;
+    type WeightInfo = pallet_treasury::weights::SubstrateWeight<Runtime>;
+    type MaxApprovals = MaxApprovals;
+}
+
+impl pallet_bounties::Config for Runtime {
```
Comment (kaichaosun) [inline runtime/parallel/src/lib.rs:853]: We probably won't use bounties at launch.

File: runtime/parallel/src/lib.rs (hunk 3 above)
Comment (ayushmishra2005) [reply runtime/parallel/src/lib.rs:639]: I picked this from https://github.com/paritytech/substrate/blob/3f110196163b5ec03bac5ee188d60bedf3ebd91d/bin/node/runtime/src/lib.rs#L560
This is also the same in Polkadot.

@kaichaosun please let me know what value you want me to put here

Comment (kaichaosun) [reply runtime/parallel/src/lib.rs:639]: Let's revisit and change the config later.

**Review by kaichaosun — APPROVED** [review]
👍

**Pull request merged**